### DISCUSSION TITLE
@types/aws-lambda Subject key is optional in SNSMessage interface

### DISCUSSION THREAD

**Discussion by jgoux** [DefinitelyTyped/DefinitelyTyped#54379] [Issues with a @types package] [answered]
Hello all,

According to AWS, the [`Subject`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/aws-lambda/trigger/sns.d.ts#L26) key is optional.

AWS doc : https://docs.aws.amazon.com/sns/latest/dg/sns-message-and-json-formats.html

**Comment by markscamilleri** [comment]
Just saw this - I opened a PR earlier this week to fix another issue with the SNS types so I included this in that PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65696

**Comment by markscamilleri** [comment, marked as answer]
@jgoux This has just been merged in and has been published as version `8.10.116`
